src/main/java/seedu/address/model/task/timeformat/IsoDateTime.java

```diff
@@ -17,6 +17,9 @@
     @Override
     public LocalDateTime check(String date) {
         try {
+            if (date.equals("")) {
+                return null;
+            }
```

Comment:
Would you think another class called EmptyFormat implements TimeFormat is better?
e.g. 
```suggestion
           public LocalDateTime check(String time) {
           if (time.equals("")){
           return null;
           }
           throw new TimeFormatException();
           }
```

Reply:
I was thinking that technically `""` is also a valid dateTime from the user input or reading from the storage file so it should not throw an exception :?